doc/cdi-config.md

```diff
@@ -16,7 +16,7 @@ CDI configuration in specified by administrators in the `spec.config` of the `CD
 | global                   | "0.055"       | The amount to reserve for a Filesystem volume unless a per-storageClass value is chosen.                                                                                                                                     |
 | storageClass             | nil           | A value of `local: "0.6"` is understood to mean that the overhead for the local storageClass is 0.6.                                                                                                                         |
 | preallocation            | nil           | Preallocation setting to use unless a per-dataVolume value is set                                                                                                                                                            |
-
+| ImportProxy              | nil           | updated when a CDIConfig is updated or when Cluster Wide-Proxy (Openshift) is updated. If `ImportProxy` is set, importer pod URL will be ignored and `ImportProxy.HTTPSProxy` or `ImportProxy.HTTPProxy` will be updated with the user defined URL. HTTPS has priority. If `ImportProxy.NoProxy` is set, the proxy URL will be ignored if it contains any of the listed hostnames and/or CIDRs. If `ImportProxy.HTTPSProxy` is set, the `ImportProxy.TrustedCAProxy` must be provided. |
```

Comment:
@awels I add the documentation here, but I am not sure if we need to update somewhere else.

Reply:
We need to update the overall documentation that all that stuff is now set in the CDI CR, but that is not your problem. This works for me.

Reply:
I see! Thanks!
Just to let you know, I did a last commit updating the description of importerProxy in this documentation.